test/test_warp_scan.cu

```diff
@@ -458,7 +464,7 @@ void Test(
     fflush(stdout);
 
     // Run aggregate/prefix kernel
-    WarpScanKernel<LOGICAL_WARP_THREADS, TEST_MODE><<<1, TOTAL_ITEMS>>>(
+    WarpScanKernel<LOGICAL_WARP_THREADS, TEST_MODE><<<1, LOGICAL_WARP_THREADS * TOTAL_WARPS>>>(
```

Comment:
Why was this change made? Isn't `TOTAL_ITEMS == LOGICAL_WARP_THREADS * TOTAL_WARPS`

Reply:
It is, but items being processed doesn't immediately represent the number of threads you want. Now the code is written in terms of threads, not items.